code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2010,6 +2130,11 @@
 	reagent_state = LIQUID
 	color = "#500064" //rgb: 80, 0, 100
 
+/datum/reagent/plasma/New()
+	..()
+	specheatcap = rand(1,150)/rand(1,25)
```

Comment:
Won't a random specific heat capacity and density for such a common material be frustrating, since it will cause unpredictable behavior?
Is there any way to measure the specific heat capacity and density of a given instance of reagents?

Reply:
- It's plasma. It seems to be what the game is based around that it is some form of 'magic energy source' but the game has treated it as hydrogen so far, so thought I'd give some degree of strangeness
- Chem Masters can analyse a sample and give you a brief description of the reagent along with its SHC and density